### DISCUSSION TITLE
用Uno Platform做一个安卓版

### DISCUSSION THREAD

**Discussion by oxygen-dioxide** [Richasy/Bili.Uwp#347] [想法]
Uno Platform是一个跨平台应用框架，能够让UWP在Android、ios、mac、Linux系统上运行。现在b站官方客户端太臃肿，希望能够用Uno Platform把Bili.Uwp移植到Android。

**Comment by Q-Tai-mu** [comment]
自己动手丰衣足食

**Reply by oxygen-dioxide** [reply]
但是我不会.net和visual studio开发，所以只能把这个框架分享在这里

**Reply by Q-Tai-mu** [reply]
官方有app啊
，重复造轮子了

**Comment by Richasy** [comment]
这个目前不考虑，Uno很棒，但哔哩是完全为桌面环境设计，最多加个xbox，适配手机还要做一堆体验上的东西，有兴趣的同学可以参考哔哩的实现自己做一套
